acoustid/db.py

```
@@ -1,5 +1,7 @@
+import zlib
 from typing import TYPE_CHECKING, Any, Dict, NewType, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)